### ISSUE TITLE
Add support for Markdown rendering in comments

### ISSUE THREAD

**Issue by glenux** [fatfreecrm/fat_free_crm#440]
Having support for titles/bold/italic/links rendering in notes would be awesome.

I was thinking to include some rendering system (on the JS side) like https://github.com/evilstreak/markdown-js.

Any suggestion ?

**Comment by steveyken** [comment]
Yeah markdown support would be great. I'd slightly favour doing on the
server side so that future API access (via json/xml etc) will just work but
I could be swayed either way.

On Wed, Apr 22, 2015 at 11:01 PM, Glenn Y. Rolland <notifications@github.com

**Comment by dancinglightning** [comment]
Nice idea, doing it client side get's rid of any (all?)  security issues.
There seems to be a rails wrapper for it, ready to go . . 
https://github.com/rosschapman/markdownjs-rails

**Comment by johnnyshields** [comment]
@CloCkWeRX this is a reasonable feature to add.

**Comment by ferrisoxide** [comment]
Would [trix](https://github.com/basecamp/trix) or similar WYSIWYG editor be a better option here? Markdown is fine, but might non-technical users find it a bit obscure?

**Comment by CloCkWeRX** [comment]
I'm mixed on this. Knowing the sales guy at my work - markdown? Not going to fly - a WYSIWYG editor would suit him much better.
But I'm hesitant on WYSIWYG editors in general, as past a certain point when you copy a HTML table out of an email and it gets mangled :S

(Not that a non WYSIWIG approach helps either with that)

It comes down to:
* PRs welcome and
* Will this introduce any maintenance or security considerations

**Comment by ferrisoxide** [comment]
Maybe both - ala Reddit's "fancy pants" editor that can be swapped for a markdown editor? The switch between the two could be based on user preference - either globally or per comment. 

It would require more complex code but would satisfy the needs for different types of users.

**Comment by CloCkWeRX** [comment]
https://jules.google.com/task/18240413866274779639 to see how far vibe coding gets us
